File: 05_salary_increase/python/main.py
INCREMENT_PERCENTAGE = 0.1

if __name__ == '__main__':
    print("\n--- PROGRAMA PARA AUMENTAR EL SALARIO ---\n")

    salary = float(input("Ingresa tu salario actual > "))
    base = float(input("Ingresa el salario base > "))

    if salary < 2 * base:
        increase = salary * INCREMENT_PERCENTAGE
        new_salary = salary + increase

        print(f"\nTu aumento será de: $ {increase}")
        print(f"Tu nuevo salario será de: $ {new_salary}")
    else:
        print("\nNo tienes derecho a un aumento")

File: 05_salary_increase/java/src/Main.java
import java.util.Scanner;

public class Main {
    private static final double INCREMENT_PERCENTAGE = 0.1;

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("\n--- PROGRAMA PARA AUMENTAR EL SALARIO ---\n");

        System.out.println("Ingresa tu salario actual:");
        double salary = input.nextInt();

        System.out.println("Ingresa el salario base:");
        double base = input.nextInt();

        if (salary < 2 * base) {
            double increase = salary * INCREMENT_PERCENTAGE;
            double new_salary = salary + increase;

            System.out.println("\nTu aumento será de: $ " + increase);
            System.out.println("Tu nuevo salario será de: $ " + new_salary);
        } else {
            System.out.println("\nNo tienes derecho a un aumento");
        }
    }
}